assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -66,7 +66,7 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def test_merge_dicts():
</pre><p>Please keep the same style as above. You should have a test class first. Then you define all the test cases inside.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='127'/>